numpy/core/fromnumeric.py

```diff
@@ -2033,7 +2033,8 @@ def clip(a, a_min, a_max, out=None, **kwargs):
     is specified, values smaller than 0 become 0, and values larger
     than 1 become 1.
 
-    Equivalent to but faster than ``np.maximum(a_min, np.minimum(a, a_max))``.
+    Equivalent to but faster than ``np.maximum(a_min, np.minimum(a, a_max))``
+    only if ``a_min < a_max`` else returns ``a_max``.
```

Comment:
I'd be surprised if it were this straightforward, but happy to be wrong :)

I though it was more like if `amin < amax`, strange things can happen depending on the nature of the input.

Reply:
Maybe simply "Assumes `a_min < a_max`"

Reply:
I think it actually _is_ this straightforward (whereas it didn't use to be). Bring deliberately vague here is probably good anyway though.